crypto/fipsmodule/bn/asm/x86_64-mont.pl

```diff
@@ -105,8 +105,10 @@
 	jb	.Lmul_enter
 ___
 $code.=<<___ if ($addx);
+#ifndef MY_ASSEMBLER_IS_TOO_OLD_FOR_512AVX
```

Comment:
I have similar questions here and in other files where `$addx` is controlled by the flag, what is the need to add the macros?

Reply:
As above, it's only needed so that the pre-generated assembly can be used with GCC 4.8.x.